src/components/network/network-panel.html

```diff
@@ -4,6 +4,13 @@
     <button class="btn btn-default glyphicon glyphicon-refresh btn-update"></button>
     <input class="form-control input-with-update">
   </div>
+  <div id="input-type" class="panel-section">
+    <div class="item-title">Input Type</div>
+      <div class="input-checkbox" id="gene-regex-string">
+        <label class="radio-inline"><input type="radio" name="optradio">regex</label>
```

Comment:
name here is not used. Change to meaningful names/roles and select by name/role.